packages/core/src/analytics/integrations/Omnitracking/definitions.js

```diff
@@ -502,6 +503,19 @@ export const pageEventsMapper = {
     viewSubType: 'Listing',
     lineItems: getProductLineItems(data),
   }),
+  [pageTypes.WISHLIST]: data => {
+    const productLineItems = getProductLineOmnitrackingItems(data);
+
+    return {
+      viewType: 'Wishlist',
+      viewSubType: 'Wishlist',
+      lineItems: JSON.stringify(productLineItems),
```

Comment:
Can you call `getProductLineItems` here, without the `getProductLineOmnitrackingItems` call above?